### PR TITLE
feat(aapd-74): filter appeals by status, and rename from lpa-dashboar…

### PR DESCRIPTION
**Pull request by edpackard-work**
…d-appeals to appeals-case-data

## Ticket Number

https://pins-ds.atlassian.net/browse/AS-

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/appeals-service-api/__tests__/unit/services/appeals-case-data.service.test.js [hunk 1]
```diff
@@ -49,7 +50,15 @@ describe('src/services/lpa-dashboard-appeals.service.test', () => {
...
 		const result = await getAppeals(lpaCode);
 
-		expect(collectionMock.find).toHaveBeenCalledWith({ LPACode: lpaCode }, appealsProjection);
+		expect(collectionMock.find).toHaveBeenCalledWith(
+			{
+				LPACode: lpaCode,
+				validity: 'Valid',
+				questionnaireDueDate: { $regex: regEx },
+				questionnaireReceived: { $not: regEx }
+			},
```
Comment (edpackard-work) [inline packages/appeals-service-api/__tests__/unit/services/appeals-case-data.service.test.js:59]: initially tried an 'and' for due date - **is not empty or null && matches regex**, and an 'or' for received **is empty or null || does not match regex** but I don't think mongo does short circuit evaluation, so this seems pointless.

File: data/data/appeals-service-api/004-appealsCaseData.json [hunk 2]
```diff
@@ -88,7 +88,7 @@
 		"LPAStatementSubmitted": "2023-07-07T13:53:31.6003126+00:00",
 		"proofsOfEvidenceDueDate": "2023-07-07T13:53:31.6003126+00:00",
 		"questionnaireDueDate": "2023-07-07T13:53:31.6003126+00:00",
-		"questionnaireReceived": "2023-07-07T13:53:31.6003126+00:00",
+		"questionnaireReceived": null,
```
Comment (edpackard-work) [inline data/data/appeals-service-api/004-appealsCaseData.json:91]: have set this to null or data would no longer be returned

File: packages/appeals-service-api/__tests__/unit/services/appeals-case-data.service.test.js (hunk 1 above)
Comment (edpackard-work) [reply packages/appeals-service-api/__tests__/unit/services/appeals-case-data.service.test.js:58]: have replaced with checking for date type

File: packages/appeals-service-api/__tests__/unit/services/appeals-case-data.service.test.js [hunk 3]
```diff
@@ -49,7 +49,16 @@ describe('src/services/lpa-dashboard-appeals.service.test', () => {
 
 		const result = await getAppeals(lpaCode);
 
-		expect(collectionMock.find).toHaveBeenCalledWith({ LPACode: lpaCode }, appealsProjection);
+		expect(collectionMock.find).toHaveBeenCalledWith(
+			{
+				LPACode: lpaCode,
+				appealType: 'Householder (HAS) Appeal',
```
Comment (edpackard-work) [inline packages/appeals-service-api/__tests__/unit/services/appeals-case-data.service.test.js:55]: have added `appealType` field to query

File: packages/appeals-service-api/src/services/appeals-case-data.service.js [hunk 4]
```diff
@@ -18,11 +18,19 @@ const getAppeals = async (lpaCode) => {
...
 	try {
-		//todo: add required 'status' field to .find (aapd-74)
 		const cursor = await mongodb
 			.get()
 			.collection('appealsCaseData')
-			.find({ LPACode: lpaCode }, appealsProjection);
+			.find(
+				{
+					LPACode: lpaCode,
+					appealType: 'Householder (HAS) Appeal',
```
[the remark this answers is not in the document]
Comment (bryn-pins) [reply packages/appeals-service-api/src/services/appeals-case-data.service.js:27]: eh let's just get in to not hold up other work

**Review by bryn-pins — DISMISSED** [review]

File: packages/appeals-service-api/src/services/appeals-case-data.service.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (edpackard-work) [reply packages/appeals-service-api/src/services/appeals-case-data.service.js]: it is done

**Review by bryn-pins — APPROVED** [review]

**Pull request merged**